lib/instance-validator.js

```diff
@@ -334,4 +346,4 @@ InstanceValidator.prototype._pushError = function(isBuiltin, errorKey, rawError)
   error[InstanceValidator.RAW_KEY_NAME] = rawError;
 
   this.errors.push(error);
-};
+};
```

Comment:
These changes were to handle argument changes to validator module. They were copied from the fix that was made in the later version of the upstream code.